src/components/containers/visibility.tsx

```diff
@@ -64,6 +64,20 @@ const deviceRenderer = (): boolean => {
     return is_loaded
 }
 
+export const getCountryRule = () => {
+    const { is_eu_domain, is_uk_domain } = domainBasedCheck()
+    const { is_eu_country, is_uk_country } = React.useContext<StoreDataType>(DerivStore)
+
+    const is_eu = (is_eu_country || is_eu_domain) && !is_uk_country
+    const is_uk = is_uk_country || is_uk_domain
+    const is_non_uk = !is_uk
+    const is_non_eu = !is_eu
+    const is_eu_uk = !(!is_eu && !is_uk)
```

Comment:
```suggestion
    const is_eu_uk = is_eu || is_uk
```

Reply:
tried this but for some reason it is not working properly.